proxy/http2/Http2ClientSession.cc

```diff
@@ -325,6 +330,7 @@ Http2ClientSession::main_event_handler(int event, void *edata)
   case VC_EVENT_INACTIVITY_TIMEOUT:
   case VC_EVENT_ERROR:
   case VC_EVENT_EOS:
+    this->set_dying_event(event);
```

Comment:
this not called anywhere else in code. Should it be?

Reply:
No, all the dying eventually comes through this handler.